scripts/build_erddap_catalog.py

```diff
@@ -155,12 +185,33 @@ def build_erddap_catalog_fragment(data_root, user, deployment, template_dir,
             for identifier, mod_attrs in extra_atts.iteritems():
                 add_extra_attributes(tree, identifier, mod_attrs)
 
+
+            # append latest changes to etree
+
+            for var in standard_name_vars:
+                var_elem = add_standard_name_var(var)
+                tree.append(var_elem)
+
             return etree.tostring(tree)
 
         except:
-            logger.exception()
+            import ipdb; ipdb.set_trace()
+            logger.exception("Exception occurred while generating dataset XML:")
             return templ
 
+def add_standard_name_var(var):
+    """
+    Adds an unhandled standard name variable to the ERDDAP datasets.xml
+    """
+    dvar_elem = etree.Element('dataVariable')
+    source_name = etree.SubElement(dvar_elem, 'sourceName')
+    source_name.text = var.name
```

Comment:
Copying over the variable name works but we may want to revisit this to try to have consistency in the variable names.

Reply:
What are you proposing here?

Reply:
I was going to propose using the standard name but after discussing with @kerfoot we should just leave it as you have here for now.